service-manual/measurement/user-satisfaction.md

```diff
@@ -38,7 +38,7 @@ You must prompt users to give feedback at service endpoints. In the example abov
 
 You must also allow users to give feedback from anywhere in the service, in case anything goes wrong.
 
-And you must be able to show that you’re collecting user satisfaction data appropriately at your [Service Assessment](https://www.gov.uk/service-manual/digital-by-default/assessments-at-gds.html).
+And you must be able to show that you’re collecting user satisfaction data appropriately at your [Service Assessment](/service-manual/digital-by-default/assessments-at-gds.html).
```

Comment:
For example, [this post about Verify](https://gds.blog.gov.uk/2015/09/23/why-gov-uk-verify-matters/) contains 3 instances of a sentence starting with _And_.